docker/linux/Dockerfile

```diff
@@ -0,0 +1,31 @@
+FROM philcryer/min-jessie:latest
```

Comment:
Sorry for not replying earlier, I was busy with IRL stuff. Thanks for the testing & feedback ... I will work those fixes / improvements into the PR 👍